printer_mods/ppak10/Cable_Port_Flush_Mount_Skirt/README.md

```diff
@@ -0,0 +1,28 @@
+# Cable Port Flush Mount Skirt
+
+Provides an inlet to mount BATIGE circular cable adapter ports on the rear
+center skirt (250, 300, and 350) allowing easy access to display and USB headers
+of the raspberry pi.
+
+It can be used for quick access to Raspberry Pi ports often out of reach
+underneath the printer without needing to flip over the printer.
+
+![250_300_350_Render](./Images/250_300_350_render.png)
```

Comment:
Much nicer, this one is still broken though: the `R` in `render` should be capitalized.